test/suites/storage_driver_zfs.sh

```diff
@@ -325,12 +237,51 @@ do_storage_driver_zfs() {
   lxc exec c4 -- test -f /root/foo
   ! lxc exec c4 -- test -f /root/bar || false
 
-  lxc storage set lxdtest-"$(basename "${LXD_DIR}")" volume.size=1GiB
+  # Snapshot restore after cloning
+  # Create container with multiple snapshots
+  lxc launch testimage c8
+  lxc exec c8 -- touch /root/file1
+  lxc snapshot c8 snap1
```

Comment:
LXD automatic snapshot would be `snap0` so for clarity, I'd either rely on automatic snapshot names or mimic the numbering scheme.